html_app/scb_ex/addmultipledialog2.soy

```diff
@@ -12,72 +12,62 @@
         Add Samples </h1>
 
     <div class='scb_ex_inner_dialog_body'>
-
-        <table class="scb_s_experiment_setup_table" role='grid' aria-label='Cell Lines'>
-
-
-
-//            {if $assignment.id == 'scb_ex2'}
-
-            <thead class="scb_s_experiment_setup_table_head">
-
-            {foreach $heading in $template.ui.add_multiple_dialog.headings}
-            <td role='columnheader' aria-label='{$heading}' class='scb_s_experiment_setup_table_heading'>
-                {$heading}
-            </td>
-            {/foreach}
+        <table class="scb_s_experiment_setup_table" role='grid' >
+            <thead class="scb_s_experiment_setup_table_head" >
+                {foreach $heading in $template.ui.add_multiple_dialog.headings}
+                <td role='columnheader' aria-label='{$heading}'
+                    class="scb_s_experiment_setup_table_heading
+                    {if $heading == 'Strain'} scb_s_experiment_setup_table_heading_strain{/if}
+                    {if $heading == ''} scb_s_experiment_setup_table_heading_checkbox{/if}" >
+                    {$heading}
+                </td>
+                {/foreach}
```

Comment:
I don't know the history behind that if statement and why it was written in the first place, but I went through the logic and the structure that it is parsing and it seems that such values as 'header' and 'order' will not appear as a value of cell_line.